packages/host/app/components/operator-mode/container.gts

```diff
@@ -149,13 +159,22 @@ export default class OperatorModeContainer extends Component<Signature> {
       @boxelModalOverlayColor='var(--operator-mode-bg-color)'
     >
       <CardCatalogModal />
-      {{#if this.isCodeMode}}
-        <CodeSubmode
-          @saveSourceOnClose={{perform this.saveSource}}
-          @saveCardOnClose={{perform this.write}}
-        />
+      {{#if this.loadMatrix.isRunning}}
+        <div class='loading'>
```

Comment:
<img width="276" alt="image" src="https://github.com/cardstack/boxel/assets/273660/7c9eea02-ad65-46f5-b3b0-ac44d4b6abef">

I get a flicker with this text on top left for half a second - aesthetically it's not the most pleasing thing. Maybe we can get away by simply not showing this at all?

Reply:
We need to tell users that they have to wait for `monacoService` to be ready before they are able to do any actions in sign-in screen. @jurgenwerk @lukemelia Do you have any advice how to address this? Maybe we can just show sign-in screen but all fields and buttons are disabled.

Reply:
One thing we could do is to make it appear long enough for teh user to read. We could do a `Promise.all()` (or ember concurrency `all()` as the case may be), and include a 500ms pause so that at the very least the message appears for 500ms. this is what we did with the auto-save message

Reply:
oh and also we should probably center the message. a flex-box works really nice for this

Reply:
It looks like this now. [c1c4410](https://github.com/cardstack/boxel/pull/872/commits/c1c441062c7364db0833f191deabdf1638804e77)


https://github.com/cardstack/boxel/assets/12637010/9741674d-532e-4278-9f48-5b171e253f59